src/DataViews/SamplingReportDataViewTest.cpp

```diff
@@ -216,17 +216,30 @@ std::string GetExpectedToolTipByIndex(size_t index, int column) {
         kStackEventsCount, percentage, percentage - kConfidenceIntervalLeftSectionLength * 100.0f,
         percentage + kConfidenceIntervalRightSectionLength * 100.0f);
   }
+  if (column == kColumnUnwindErrors) {
+    const float percentage = kSampledUnwindErrorPercents[index];
+    return absl::StrFormat(
+        "Stack containing the function \"%s\"\n"
```

Comment:
I'd like @dpallotti to provide input on the tooltip string.

Reply:
Right, in general it looks good, buuuut one clarification: the unwind errors consider the function _at the top_ of the stack (we don't know what functions are further down because, well, we couldn't unwind). "Containing" suggests the opposite, so I think we should be more precise. The sentence probably needs to be turned around a bit for that. I'm thinking:
"%u samples with the function \"%s\" at the top of the stack could not be unwound in a total of %u stack samples."